### PR TITLE
Added Here I Stand Board game

### PR DESCRIPTION
**Pull request by parthrc**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2202,6 +2202,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+
+![Here I Stand](https://cf.geekdo-images.com/SaJz1vt6j4njvkhDaplSHA__itemrep/img/28KXJz7JsY47i1eKS6TRY_elzYg=/fit-in/246x300/filters:strip_icc()/pic353047.jpg)
+
+| Players | Min. Age | Time   |
+| ------: | -------: | -----: |
+|   2 - 6 |       12 |90-360m |
```
Comment (edm00se) [inline readme.md:2213]: The stats do not match BGG, min time they list as 180 minutes.

```suggestion
|   2 - 6 |       12 |180-360m |
```

**Review by edm00se — CHANGES_REQUESTED** [review]
Please correct the stats block.

**Comment by parthrc** [comment]
Corrected the stats block

**Review by edm00se — APPROVED** [review]
refs #269

**Pull request merged**